sdk/dapp-kit/src/constants/walletDefaults.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { WalletWithRequiredFeatures } from '@mysten/wallet-standard';
+
+import { createInMemoryStore } from '../utils/stateStorage.js';
+
+export const SUI_WALLET_NAME = 'Sui Wallet';
+
+export const DEFAULT_STORAGE = typeof window === 'undefined' ? createInMemoryStore() : localStorage;
```

Comment:
should we do something type `typeof window?.localStorage` here?  I know some envs will have a fake partial window object

Reply:
or `window?.localStorage ?? createInMemoryStore()`

Reply:
If window is undefined does that identifier access throw?

Reply:
yes... it looks like it does.  I expected that the `?.` would make this safe, but it does not.  probably need to do both the typeof and then the check for localStorage

Reply:
I could use this bad boy, but that might be a little overkill 😆.

<img width="1417" alt="image" src="https://github.com/MystenLabs/sui/assets/7453188/82713103-05b8-4a37-9c3f-961a982f98e4">

 I'll update this with the following, thanks for the call out!

```
 typeof window !== 'undefined' && window.localStorage
```